content/documentation/rpc/wallet/create.mdx

```diff
@@ -3,6 +3,8 @@ title: wallet/create
 description: RPC Wallet | Iron Fish Documentation
 ---
 
+## This API has been deprecated. Please use [wallet/createAccount](../wallet/createAccount) instead.
```

Comment:
nit - I wonder if there's something we can use instead of a heading to highlight this. With a heading it's automatically included in the contents sidebar on the right:
<img width="1398" alt="image" src="https://github.com/iron-fish/website/assets/57735705/75936018-3d94-4c77-ada5-1f452bf43dc7">

Reply:
made it a simpler bold message for now. I also created a ticket for skylar to create a component for this. We can get this in and change to the component later when it's ready